packages/user/src/services/invite.ts

```diff
@@ -0,0 +1,110 @@
+import { Context, DAL } from "@medusajs/types"
+import {
+  InjectTransactionManager,
+  MedusaError,
+  ModulesSdkUtils,
+} from "@medusajs/utils"
+import { Invite } from "@models"
+import { InviteServiceTypes } from "@types"
+import jwt, { JwtPayload } from "jsonwebtoken"
+
+type InjectedDependencies = {
+  inviteRepository: DAL.RepositoryService
+}
+
+// 7 days
+const DEFAULT_VALID_INVITE_DURATION = 1000 * 60 * 60 * 24 * 7
+
+export default class InviteService<
```

Comment:
suggestion: add a `getOption(key)` method that throws if `this.options` is not defined and otherwise returns `this.options[key]`. Will give better error messages if this service is accidentally used without first calling `withModuleOptions`.